File: Data Structures & Algorithms/remove-element/submission-0.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        
        count = 0
        for n in nums:
            if n == val:
                count += 1

        while count != 0:
            nums.remove(val)
            count -= 1

        return len(nums)

File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {

    // time: O(n)
    // space: O(n)

    public boolean hasDuplicate(int[] nums) {
        Set<Integer> hashSet = new HashSet<>();

        for(int n : nums) {
            if(hashSet.contains(n)) {
                return true;
            }
            hashSet.add(n);
        }

        return false;
    }
}
